docs/user/user.yaml

```diff
@@ -259,3 +276,169 @@ paths:
+      description: Finds and returns a user`s profile information.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/user'
+              example:
+                _id: 6255bc080a75adf9223df100
+                role: student
+                firstName: John
+                lastName: Doe
+                email: johndoe@gmail.com
+                categories:
+                  [ { _id: 6255bc080a75adf9223df444, name: Languages }, { _id: 6255bc080a75adf443d7644, name: Biology } ]
+                totalReviews: { student: 10, tutor: 0 }
+                averageRating: { student: 4.5, tutor: 0 }
+                nativeLanguage: english
+                address: { country: The USA, city: California }
+                education: KNPU H.S. Skovoroda
+                photo: john-doe-photo.jpg
+                status: { student: active, tutor: active, admin: active }
+                isEmailConfirmed: true
+                isFirstLogin: true
+                lastLogin: 2022-09-02T11:59:53.243+00:00
+                bookmarkedOffers: [ ]
+                createdAt: 2021-04-09T11:34:53.243+00:00
+                updatedAt: 2022-09-02T11:59:53.243+00:00
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: 'INVALID_ID'
+                message: 'ID is invalid.'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 404
+                code: 'UNAUTHORIZED'
+                message: 'The requested URL requires user authorization.'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 404
+                code: 'USER_NOT_FOUND'
+                message: 'User with the specified email or id was not found.'
+  /users/{id}/reviews:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Users
+      summary: Find all reviews for a user with the specified ID
+      description: Finds and returns an array with a list of reviews for a user with the specified ID.
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: role
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: rating
+          schema:
+            type: integer
+          required: false
```

Comment:
is that possible to pass here default option?
to show what we have there by default?